packages/synapse-react-client/src/components/CitationPopover/CitationPopover.tsx

```diff
@@ -0,0 +1,325 @@
+        fill="#395979"
+      />
+    </svg>
+  )
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setCitationFormat(event.target.value)
+  }
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const handleDownload = (citation: string) => {
+    if (!citation) {
+      return
+    }
+    const extensionMap: Record<string, string> = {
+      apa: 'ris',
+      bibtex: 'bib',
+      ieee: 'ris',
+      nature: 'ris',
+      science: 'ris',
+    }
+    const citationTitle = title ? `${title.replace(/\s+/g, '_')}` : 'citation'
+    const fileExtension = extensionMap[citationFormat] || 'txt'
+
+    const blob = new Blob([citation], { type: 'text/plain' })
+    const url = URL.createObjectURL(blob)
+    const link = document.createElement('a')
+    link.href = url
+    link.setAttribute('download', `${citationTitle}.${fileExtension}`)
+    document.body.appendChild(link)
+    link.click()
+    document.body.removeChild(link)
+  }
+
+  const handleCopy = (citation: string) => {
+    if (!citation) {
+      return
+    }
+    navigator.clipboard.writeText(citation)
+  }
+
+  const fetchCitation = async (doi: string | undefined, format: string) => {
+    const formattedDoi = doi ? doi.replace('https://doi.org/', '') : ''
+    try {
+      const response = await fetch(
+        `https://citation.doi.org/format?doi=${formattedDoi}&style=${format}&lang=en-US`,
+        {
+          headers: {
+            Accept: `text/x-${format}`,
+          },
+        },
+      )
+      if (!response.ok) {
+        throw new Error(
+          `Failed to fetch citation: ${response.status}: ${response.statusText}`,
+        )
+      }
+      return await response.text()
+    } catch (err) {
+      console.log(err)
+      return ''
+    }
+  }
+
+  const open = Boolean(anchorEl)
+  const id = open ? 'cite-as-popover' : undefined
+
+  const useCitation = (doi: string, format: string) => {
+    return useQuery({
+      queryKey: ['citation', doi, format],
+      queryFn: () => fetchCitation(doi, format),
+      enabled: !!doi && !!format && !!open,
+    })
+  }
+
+  const { data: citation, isLoading } = useCitation(doi || '', citationFormat)
+
+  return (
+    <div>
+      {doi && (
+        <Button
+          onClick={handleClick}
+          sx={{
+            display: 'flex',
+            height: '20px',
+            padding: '2px 8px',
+            justifyContent: 'center',
+            alignItems: 'center',
+            gap: '4px',
+            textDecoration: 'none !important',
+            borderRadius: '3px',
+            border: '1px solid',
+            borderColor: 'grey.400',
+          }}
+        >
+          <Box sx={{ display: 'flex' }}>{quotationSvg}</Box>
+          <Typography variant="label" sx={{ lineHeight: '20px' }}>
+            Cite As
+          </Typography>
+        </Button>
```

Comment:
Use props `variant='outlined'` and `startIcon={quotationSvg}` and remove extraneous styles